log-viewer/out/index.html

```diff
@@ -5,12 +5,8 @@
 		<script type="text" id="LOG_FILE_NAME">@@name</script>
 		<script type="text" id="LOG_FILE_PATH">@@path</script>
 		<script type="text" id="LOG_FILE_NS">@@ns</script>
-		<script type="text" id="LOG_FILE_TXT">
-		  @@logTxt
-        </script>
-		<script type="text/javascript">
-		  @@bundle
-		</script>
+		<script type="text" id="LOG_FILE_URI">sample.log</script>
```

Comment:
In prod, it's just a placeholder but when run standalone it will try and load sample.log so you can use that for setting up test cases. I was pondering about adding drop support which might be nice in either env. The problem you have in browser testing at the moment is that there is no other way to load a log.

Reply:
Maybe I should stick a comment on that ;-)